src/SB/Core/x/xParEmitter.cpp

```diff
@@ -1,3 +1,828 @@
 #include "xParEmitter.h"
 
 #include <types.h>
+#include <zScene.h>
+#include <iModel.h>
+#include <xMathInlines.h>
+#include <xMath.h>
+#include <xGroup.h>
+#include <zGlobals.h>
+#include <xDebug.h>
+
+void add_tweaks(xParEmitter& pe)
+{
+}
+
+S32 xParInterpConvertInterpMode(xParInterp* p)
+{
+    U32 interp_val;
+
+    interp_val = p->interp;
```

Comment:
Let's do the declaration and assignment on the same line since the variable is initialized immediately.